src/components/pagination/pagination.hbs

```diff
@@ -2,9 +2,9 @@
   <div class="bx--pagination__left">
     <div class="bx--select{{#is version "v2"}} bx--select--inline{{/is}}">
       {{#is version "v2"}}
-        <label for="select-id-pagination" class="bx--pagination__text">Items per page: </label>
+        <label for="items-per-page" class="bx--pagination__text">Items per page: </label>
```

Comment:
Is this the correct label? `<label>` content should be the same as `aria-label` for `select` if we are using the `for` attribute, I thought? Let me know if I'm off here 🤔

Reply:
Totally adding redundancy here, you're right. There's only an implicit label in version 2, so the added aria-label is DAP errors in other versions.